content/blog/joining-ggplot2/index.Rmd

```diff
@@ -0,0 +1,118 @@
+* [ ] Create `thumbnail-sq.jpg`; height and width should be equal
+* [ ] Create `thumbnail-wd.jpg`; width should be >5x height
+* [ ] `hugodown::use_tidy_thumbnails()`
+* [ ] Add intro sentence, e.g. the standard tagline for the package
+* [ ] `usethis::use_tidy_thanks()`
+-->
+
+Hello there! I've been working on ggplot2 for a while now, and I'd like to tell you how that came about and what it is like.
+
+## How I got involved
+
+My journey into learning R started in 2017 during an internship at the EMBL-EBI.
+The main gripe about base R plotting that drove me into ggplot2's arms were the arcane invocations to get anything else than one of the pre-approved chart types.
+In contrast, ggplot2 absorbs a bunch of small paper cuts, is very compositional in nature while remaining highly customisable.
+In a bid to "learn from the mistakes of others" rather than (continue to copiously) make my own, I became active on Stack Overflow answering questions and solving plotting issues. 
+For posterity: this was in the days before you could ask an large language model for personalised advice and actual humans were equally frustrated on both sides of the question.
+
+I was keeping track of solutions to common problems in a personal cookbook that had its own arcane invocations.
+To give a bit of flavour: much of the cookbook was about preparing gtables, the data structure that comes out of building a plot, for combining and aligning plots. ^[Luckily, we don't have to think about this *at all*, thanks to the [patchwork](https://patchwork.data-imaginist.com/) package!]
+The cookbook eventually grew into my first ggplot2 extension package: [ggh4x](https://teunbrand.github.io/ggh4x/).
+Perhaps that package would be best subtitled: 'Remedies to my common ggplot2 ailments'.
+It contains a bunch of miscellaneous functions ranging from reorganising facets to putting minor ticks on the axes.
+Simultaneously this was also a downside, as ggh4x lacked any sense of scope (and still does, as befits any first package).
+
+Around the time I was really getting into ggplot extensions, Gina Reynolds had started organising a meeting for people who build ggplot2 extensions.
+It is an interesting place to meet others and hear about their packages and how they face interacting with the ggplot2 extension system.
+I started attending with some degree of regularity and made a discussion place on GitHub.
+We now use this for general exchange of ideas, but also package specific issues.
+
+Meanwhile, the questions on Stack Overflow kept directing my attention at the ggplot2 issue tracker every once in a while.
+After lurking in there for a bit, I started my first informal contributions to ggplot2 itself by answering the simple stuff just as I did on Stack Overflow.
+It may not seem like much of a contribution, but in retrospect, answering issues helps triaging them: it divorces those issues who need additional changes in ggplot2 from those that do not.
```

Comment:
```suggestion
It may not seem like much of a contribution, but in retrospect, answering issues helps triaging them: it separate those issues that need additional changes in ggplot2 from those that do not.
```